app/components/school-vocabularies-list.hbs

```diff
@@ -1,170 +1,104 @@
-            @action={{this.toggleShowNewVocabularyForm}}
-          />
-        </div>
-      </div>
-    {{/if}}
-    <section class="school-vocabularies-list-new">
-      {{#if this.showNewVocabularyForm}}
-        <div class="form" data-test-new-vocabulary-form>
-          <div class="item" data-test-title>
-            <label for="title-{{templateId}}">
-              {{t "general.title"}}:
-            </label>
-            <input
-              id="title-{{templateId}}"
-              type="text"
-              value={{this.newVocabularyTitle}}
-              {{on "input" (pick "target.value" (set this.newVocabularyTitle))}}
-              {{on "keyup" (queue (fn this.addErrorDisplayFor "newVocabularyTitle") (perform this.saveOrCancel))}}
-              data-test-title
-            >
-            {{#each (await (compute this.getErrorsFor "newVocabularyTitle")) as |message|}}
-              <span class="validation-error-message">
-                {{message}}
-              </span>
-            {{/each}}
-          </div>
-          <div class="buttons">
-            <button
-              type="button"
-              class="done text"
-              data-test-submit
-              {{on "click" (perform this.saveNew)}}
-            >
-              {{#if this.saveNew.isRunning}}
-                <LoadingSpinner />
-              {{else}}
-                {{t "general.done"}}
-              {{/if}}
-            </button>
-            <button type="button" class="cancel text" data-test-cancel {{on "click" this.toggleShowNewVocabularyForm}}>
-              {{t "general.cancel"}}
-            </button>
-          </div>
-        </div>
-      {{/if}}
-      {{#if this.newVocabulary}}
-        <div class="savedvocabulary" data-test-saved-vocabulary>
-          <span
-            class="clickable link"
-            role="button" {{on "click" (fn @manageVocabulary newVocabulary.id)}}
-            data-test-manage-new-vocabulary
-          >
-            <FaIcon @icon="external-link-square-alt" />
-            {{this.newVocabulary.title}}
-          </span>
-          {{t "general.savedSuccessfully"}}
-        </div>
-      {{/if}}
-    </section>
-    <div class="school-vocabularies-list-list">
-      {{#if this.load.isRunning}}
-        <LoadingSpinner />
-      {{else}}
-        {{#if this.sortedVocabularies.length}}
-          <table>
-            <thead>
-              <tr>
-                <th class="text-left" colspan="3">
-                  {{t "general.vocabulary"}}
-                </th>
-                <th class="text-left" colspan="1">
-                  {{t "general.terms"}}
-                </th>
-                <th class="text-left" colspan="1">
-                  {{t "general.actions"}}
-                </th>
+<div
+  class="school-vocabularies-list"
+  data-test-school-vocabularies-list
+  {{did-insert (perform this.load)}}
+  {{did-insert (perform this.load) @school.vocabularies}}
```

Comment:
This should be `did-update`, but would be even better replaced with something like https://github.com/ilios/frontend/blob/050667fbcdd103f9a1a189b910c144d366f40d28/app/components/program-year/competencies.js#L16

Reply:
fixed :heavy_check_mark: